lib/elixir/pages/Writing Documentation.md

```diff
@@ -53,33 +53,41 @@ The compiler will infer this argument as `map`. Sometimes the inference will be
 
 ## Recommendations
 
-There are a couple tips we recommend developers to follow when writing documentation:
+When writing documentation:
```

Comment:
I prefer the shorter version. As we say on the line below: "Keep the first paragraph of the documentation concise and simple".